charts/sentry/templates/sentry/_helper-sentry.tpl

```diff
@@ -195,8 +195,10 @@ sentry.conf.py: |-
   # information on configuring your queue broker and workers. Sentry relies
   # on a Python framework called Celery to manage queues.
 
-  {{- if or (.Values.rabbitmq.enabled) (.Values.rabbitmq.host) }}
```

Comment:
This is not backwards compatible, my deployment relies on this to get the external rabbit running